rj_common/include/rj_common/field_dimensions.hpp

```diff
@@ -213,7 +213,7 @@ struct FieldDimensions {
     float goal_height_;
     float penalty_short_dist_;
     float penalty_long_dist_;
-    float arc_radius_;
+    //float arc_radius_;
```

Comment:
delete this line, don't just comment it out. we can always revert the commit if we want this code back :)